Source/WebKit/UIProcess/mac/WKTextIndicatorStyleManager.h

```diff
@@ -37,8 +37,6 @@ class WebViewImpl;
 
 @interface WKTextIndicatorStyleManager : NSObject
 
-+ (BOOL)supportsTextIndicatorStyle;
```

Comment:
Just to confirm: This is being removed as we no longer need to worry about the framework being missing on latest macOS right?

Reply:
Yes, it should always be there/be guarded in other ways, this was for dealing with the soft-linking, etc, while in development.